### DISCUSSION TITLE
'Target overrides' section in REAME.md

### DISCUSSION THREAD

**Discussion by bessarabov** [umputun/spot#53] [General]
Now in REAME.md there is a text:

```
Target overrides
There are several ways to override or alter the target defined in the playbook file:

--inventory set hosts from the provided inventory file or url. Example: --inventory=inventory.yml or --inventory=http://localhost:8080/inventory.
```

I don't understand it. In the header in the first line there is info about `target`, but in the list there are several options (not only target). Probably something is wrong with the title/text

**Comment by umputun** [comment]
Yeah, this is about overrides related to the target. Inventory is sort of related ... so I have it here. But feel free to make it more clear
